evennia/commands/default/admin.py

```diff
@@ -76,12 +76,11 @@ def func(self):
             # Boot by account object
             pobj = search.account_search(args)
             if not pobj:
-                caller.msg("Account %s was not found." % args)
+                caller.msg(f"Account {args} was not found.")
                 return
             pobj = pobj[0]
             if not pobj.access(caller, "boot"):
-                string = "You don't have the permission to boot %s." % (pobj.key,)
-                caller.msg(string)
+                caller.msg(f"You don't have the permission to boot {pobj.key,}.")
```

Comment:
This looks like it will traceback. You need to remove the comma from `{obj.key,}`.